src/anomalib/models/draem/lightning_model.py

```diff
@@ -145,6 +145,8 @@ def configure_callbacks(self) -> list[EarlyStopping]:
         )
         return [early_stopping]
 
-    def configure_optimizers(self) -> torch.optim.Optimizer:
+    def configure_optimizers(self) -> tuple[list[torch.optim.Optimizer], list[torch.optim.lr_scheduler.LRScheduler]]:
         """Configure the Adam optimizer."""
-        return torch.optim.Adam(params=self.model.parameters(), lr=self.hparams.model.lr)
+        optimizer = torch.optim.Adam(params=self.model.parameters(), lr=self.hparams.model.lr)
+        scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer, milestones=[400, 600], gamma=0.1)
+        return [optimizer], [scheduler]
```

Comment:
We previously did not add this because early stopping is enabled, and the training rarely reaches 400 epochs. But since we recently decided to always use the configuration recommended in the paper as the default, I agree it would be good to add it now.

In this case we should probably also disable early stopping by default.

Reply:
From a few training runs I did, the last few epochs make a little difference, but it is noticeable. With current design, removing early stopping would also require changing of this function:
https://github.com/openvinotoolkit/anomalib/blob/66532fc1625020ce7ff6f3fc1565663aa540e834/src/anomalib/models/draem/lightning_model.py#L132-L146
Should I then remove it from config and also add an if statement here to check if it's present or should it be removed altogether.

Reply:
I think the user should have the possibility to enable early stopping if they want to, so let's add a check

Reply:
Okay, this is now implemented.